app/views/liquids/_jsitor.html.erb

```diff
@@ -0,0 +1,8 @@
+<iframe
+  height="<%= height %>"
+  src="<%= link %>"
+  scrolling="no"
+  frameborder="no"
+  allowtransparency="true"
+  style="width: 100%;">
```

Comment:
could you add `loading="lazy"` to this? Since there's a PR going around adding lazy loading to iframes, we might as well get that in right away here :D

Reply:
ohh. hey sir is that the right behavior? @ashvin777. 

@rhymes ill add it when I get back home :smile: also are my test suite fine? I dont know how to test that kind of behavior, but in Integration test maybe?

Reply:
@artoodeeto I think we should be fine without testing the "defaultness" of jsitor when the id is incorrect

Reply:
Thank @rhymes for noticing this issue. I will fix this behavior. Ideally, it should show a "404" error page.

Reply:
I think the current behavior is fine as well, I'll let you evaulate how hard it'd be to intercept something like that

Reply:
@ashvin777 @rhymes we could check the response status? make an HTTP request on the backend check status if 2** then continue loading the page 4** then raise an error? what do you think?

Reply:
Don't think so.

```shell
➜  ~ curl -I https://jsitor.com/foobar
HTTP/1.1 200 OK
➜  ~ curl -I https://jsitor.com/embed/foobar
HTTP/1.1 200 OK
```

I'm thinking we shoulnd't get stuck on this detail, it likely won't matter much because the publisher will likely check if their embed is correct before publishing.

As long as there are no security fallacies here I don't think spending time to intercept if the ID is correct or not is worth it.

Reply:
@rhymes  I agree with you. also we have the guide to "guide" them lols. but we could technically do it if @ashvin777 will update the site to return 400.

Reply:
heres my fix for lazy loadig. 😄 @rhymes